### DISCUSSION TITLE
Consider adding metNames to the metabolites.tsv annotation file

### DISCUSSION THREAD

**Discussion by JonathanRob** [SysBioChalmers/Human-GEM#399] [Ideas]
I realize that adding metabolite names (`metNames`) to the `metabolites.tsv` file would duplicate the information already contained within the model file, but I think it makes sense to have a human-readable metabolite name present directly within the annotation file for convenience.

Likewise, we could also consider adding `rxnNames` to the `reactions.tsv` annotation file, though since many of these are missing, I do not think it's as important.

What do others think?

**Comment by haowang-bioinfo** [comment]
it's a great idea to provide `metNames` and `rxnNames` to `metabolites.tsv` and `reactions.tsv`, respectively. 
 you have my full support @JonathanRob

**Reply by mihai-sysbio** [reply]
Conceptually, this is pushing more information into the relatively non-standard `tsv`. Before continuing in this direction, I would suggest to look at how different file formats deal with this. If I remember correctly, the COBRA-defined `yaml` is self-contained, which in our case would mean merging the entire `tsv` into the `yaml`. Alternatively there is SBtab, but I think that is a binary format, which makes it not applicable for our use case.

**Reply by JonathanRob** [reply]
One of the main motivators for the tsv file is that its format is very intuitive and familiar for people even beyond the modeling community. In my experience, the tsv files are exactly what many users are looking for, so I would generally be opposed to phasing them out.
